homeassistant/components/enphase_envoy/strings.json

```
@@ -400,6 +400,33 @@
     },
     "envoy_error": {
       "message": "Error communicating with Envoy API on {host}: {args}"
+    },
+    "not_initialized": {
+      "message": "{service}: Enphase envoy is not yet initialized"
+    },
+    "not_collected": {
+      "message": "{service}: this endpoint is not collected by the Envoy"
+    },
+    "envoy_service_error": {
+      "message": "Error in Envoy Service: {service} {args}"
+    },
+    "envoy_service_envoy_not_found": {
+      "message": "No Envoy found: {service} {args}"
+    },
+    "envoy_service_invalid_parameter": {
+      "message": "Invalid parameters passed to Envoy service: {service} {args}"
+    }
+  },
+  "services": {
+    "get_raw_tariff": {
+      "name": "Tariff details",
+      "description": "Full tariff information from cache.",
+      "fields": {
+        "config_entry_id": {
+          "name": "Select Envoy",
```

```suggestion
          "name": "Envoy entry",
```
Or something along these lines, like, `select envoy` sounds like an action, while its a label, so it should describe what we put in

---

Makes sense, implemented your suggestions.